build.gradle

```diff
@@ -177,6 +182,7 @@ subprojects {
             systemProperty 'log4j2.contextSelector', 'org.apache.logging.log4j.core.selector.BasicContextSelector'
             // Verify assertions in tests
             jvmArgs = ['-ea', '-XX:+HeapDumpOnOutOfMemoryError']
+            systemProperty "junit.jupiter.execution.timeout.default", "5m"
```

Comment:
This is a new behavior, tests taking longer than 5 minutes will fail with this timeout.  I've seen a couple of runs with no console output but seemingly there is a hung test case.

Reply:
Thanks for calling this change out.  I'm fine w/ this behavior, but would also be happy to approve another change where this read a variable, etc to allow devs w/ slower machines to override.